unlock-app/src/components/interface/keychain/Key.tsx

```diff
@@ -306,6 +316,23 @@ function Key({ ownedKey, account, network }: Props) {
                       </MenuButton>
                     )}
                   </Menu.Item>
+                  {!isKeyExpired && (
```

Comment:
We need to check more things here, as the key may _not_ be transferable. for example please check that the key owner is also the key manager (or no key manager is set), as well as that there is no 100% fee on transfers!